packages/eslint-config-godaddy-flow/package.json

```diff
@@ -29,9 +29,13 @@
     "eslint": ">=6.0.0"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.3",
+    "@babel/core": "^7.16.5",
+    "@babel/eslint-parser": "^7.16.5",
```

Comment:
is the new parser backwards compatible with eslint@6 & eslint@7?

We have this:

```json
  "peerDependencies": {
    "eslint": ">=6.0.0"
  },
```

Reply:
If the new parser is _not_ compatible with 6 & 7, we should probably have a patch release for our current version to restrict eslint to less than 8 `"eslint": ">=6.0.0 <8.0.0"`. Then include the new parser changes in a new major release.

Reply:
@msluther `babel-eslint` was [deprecated in March 2020](https://tjaddison.com/blog/2021/03/updating-babel-eslint-to-babeleslint-parser-for-react-apps/#:~:text=As%20of%20March%202020%2C%20babel,%40babel%2Feslint%2Dparser%20.) and eslint@7 was [released on May 2020](https://github.com/eslint/eslint/releases/tag/v7.0.0), so we shouldn't have an issue with compatibility for eslint@6 and 7.